packages/@uppy/companion/src/standalone/index.js

```diff
@@ -1,5 +1,6 @@
 const express = require('express')
 const qs = require('querystring')
+const URL = require('url').URL
```

Comment:
the `"engines"` field in package.json says companion supports Node.js 6.0 and up, but the `URL` class was added in 6.13.0. I think we should drop support for Node 6 but until then, perhaps we could use the old `url.parse` and `url.format` APIs, or add a fallback to use the [`whatwg-url`](https://www.npmjs.com/package/whatwg-url) package?